pkg/pool-manager/pool.go

```diff
@@ -213,6 +221,32 @@ func (p *PoolManager) isInstanceOptedIn(namespaceName, mutatingWebhookConfigName
 	return false, nil
 }
 
+func (p *PoolManager) getOptMode(mutatingWebhookConfigName, webhookName string) (OptMode, error) {
```

Comment:
maybe you can pass this function no arguments and define `mutatingWebhookConfigName` and `webhookName` as constants.   Then you could export this function (change to GetOptMode) and frop the wrapper `GetVmOptMode` in virtualmachine_pool.go

Reply:
not sure I follow.
The reason I used a generic `getOptMode()`, is that there are currently 2 webhooks: for vms, and for pods.
In this PR I'm only addressing the vm Opt mode, but in the future we may adress the same for pods, so using a generic function seems fitting.

Reply:
Does `mutatingWebhookConfigName` `webhookName string` will have different values on the pods webhook?
why those names have to passed as arguments?

Reply:
yes, they have the same mutatingWebhookConfigName, but different webhookName

Reply:
got it. Thanks!